### DISCUSSION TITLE
Problem after clean installation

### DISCUSSION THREAD

**Discussion by tizfra** [GeekFunkLabs/fluidpatcher#39] [Q&A] [answered]
Hi everyone,
after a new installation of the OS and fluidpatcher,
i try to start `./squishbox.py` and the result is
`/usr/local/lib/python3.9/dist-packages/RPLCD/gpio.py:149: RuntimeWarning: This channel is already in use, continuing anyway.  Use GPIO.setwarnings(False) to disable warnings.
  GPIO.setup(pin, GPIO.OUT)
bad config file`

I haven't modified any file.
Can anyone help me?

edit:
I've also found that `./headlesspi.py` starts without any problem.
Wich config file i neet to edit?
Thanks

**Comment by albedozero** [comment, marked as answer]
Thanks for the info - this is a bug created by the fact that _pi_ is no longer the default user in new installs of Raspberry Pi OS, so _squishbox.py_ couldn't find the config file. It's fixed in [v0.7.2](https://github.com/albedozero/fluidpatcher/releases/tag/v0.7.2), so re-running the install script should have everything working!

**Reply by tizfra** [reply]
Perfect, now squish box works like a charm
